ui/src/stuff.js

```diff
@@ -0,0 +1,18 @@
+const disqus_config = function () {
+    this.page.url = 'http://gd77.sp5drs.xyz';
+    this.page.identifier = 'gd77';
+};
+
+(function(window, d) {
+    window.dataLayer = window.dataLayer || [];
+    const gtag = () => {
+      dataLayer.push(arguments);
+    }
+    gtag('js', (new Date()).toString());
```

Comment:
What's the purpose of this?

Reply:
Ask Google Analytics, it's their code :)

Reply:
Point me to source?